lib/gui/app/components/wifi/react/keyboard/react-virtual-keyboard.custom.jsx

```diff
@@ -0,0 +1,160 @@
+'use strict'
+
+const React = require('react')
+const PropTypes = require('prop-types')
+const jQuery = require('jquery')
+const keyboard = require('virtual-keyboard/dist/js/jquery.keyboard.js')
```

Comment:
Is this mixing React & jQuery? What's the reason for this – is this a requirement of the virtual-keyboard module?

Reply:
I deleted this file, it was to circumvent that stupid `type` error I was getting